src/main/java/ui/BoardAutoCreator.java

```diff
@@ -133,15 +126,20 @@ private void createWorkAllocationBoard() {
                         "It is saved under the name \"" + boardName + "\".");
     }
 
-    private void createSampleBoard() {
+    /**
+     * Creates a sample board to showcase HubTurbo's functionality with sample repos in filters.
+     * The dialog box shows a confirmation message for the saving of the sample board if isDialogShown is set to true.
```

Comment:
The header comment says `saving of the sample board` but the dialog is not about saving the sample board right?